cmplxfoil/CMPLXFOIL.py

```diff
@@ -38,10 +38,10 @@ class is to provide an easy to use wrapper for xfoil for intergration
 # Handle plotting imports
 plt = None
 try:
-    import matplotlib as mpl
+    from matplotlib import pyplot
+    import matplotlib.lines as mpl_lines
 
-    plt = mpl.pyplot
-    mpl_lines = mpl.lines
+    plt = pyplot
 except ImportError:
```

Comment:
In my opinion the error should only be thrown if the user actually tries to plot the airfoil (the only thing that uses matplotlib). This already happens at the beginning of the `plotAirfoil` function. That way the user won't get a warning every time cmplxfoil is imported when they don't intend to use the plotting functionality.

Reply:
Sound good to me. Though is it worth moving this import to that function alone then? I am not sure what the best practice is here.

Reply:
The reason that we don't is because the `plotAirfoil` function (or someone else) might call `_updateAirfoilPlot`, which also needs the same variables (colors, plotting, etc.). If we did the import in `plotAirfoil`, those variables may not be available.

Reply:
Oh I see, the error has to be in two places that are nested, so you risk double importing if you put it there. In that case, this is fine by my.